### PR TITLE
Adds other versions of Whereabouts

### PR DESCRIPTION
**Pull request by claudiubelu**
Added support for Velero 1.12.1 and 1.9.5. Added ``rockcraft.yaml`` files based on their Dockerfiles.
 
 Depends-On: https://github.com/canonical/k8s-test-harness/pull/11
 Depends-On: https://github.com/canonical/whereabouts-rock/pull/11

### CODE REVIEW

**Review by bschimke95 — COMMENTED** [review]
Please resolve conflict

**Comment by claudiubelu** [comment]
> Please resolve conflict

Done

File: tests/sanity/test_whereabouts.py [hunk 1]
```diff
@@ -41,9 +41,23 @@ def test_whereabouts_rock():
     _run_in_docker(image, True, "ls", "-la", *ROCK_EXPECTED_FILES)
 
     # check binary name and version.
+    process = subprocess.run(
```
Comment (bschimke95) [inline tests/sanity/test_whereabouts.py]: nit: you could use the `run` from the `k8s-test-harness`

**Review by bschimke95 — APPROVED** [review]
LGTM, great work!

File: tests/sanity/test_whereabouts.py (hunk 1 above)
Comment (claudiubelu) [reply tests/sanity/test_whereabouts.py]: Done, now depends on: https://github.com/canonical/k8s-test-harness/pull/11

**Comment by bschimke95** [comment]
Only image upload to ghcr fails because of external contributor - merging.

**Pull request merged**